### PR TITLE
rudimentary websocket implementation

### LINKED ISSUE #33: Get backend data via web sockets ESTIMATE: 3h
Now its possible to receive initial state from wss://iviz-back.herokuapp.com/ instead of using initial-state.js. The JSON-object is sent to client on connect.

### PR DESCRIPTION
**Pull request by fyrkant**
This PR connects the frontend code to the backend. I'm not really satisfied with this implementation (unsure where the websocket-code should be put) but it will have to do for now. :sleepy: 

Closes #33

### CODE REVIEW

File: src/actions/websocket.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import C from "../constants";
+export const websocket = new WebSocket("ws://localhost:5000");
```
Comment (rpkarlsson) [inline src/actions/websocket.js]: You forgot something here :P

**Comment by fyrkant** [comment]
:skull: :gun:

**Comment by rpkarlsson** [comment]
Nice one. Full integration test tomorrow then!

**Comment by fyrkant** [comment]
Yeah, while the front-end now receives the data it still doesn't do diddly-squat with it, so well have stuff to do tomorrow too! :sushi: (sushi because it's tasty)

**Comment by rpkarlsson** [comment]
I know but just need to change the initial time that sphere and earth starts it timeline at and trim the loop so we get a nice time flow. Should be able to do that and write a integration test tomorrow!

**Pull request merged**